pkg/api/validation/validation.go

```diff
@@ -606,8 +606,8 @@ func validateCinderVolumeSource(cd *api.CinderVolumeSource) validation.ErrorList
 	if cd.VolumeID == "" {
 		allErrs = append(allErrs, validation.NewFieldRequired("volumeID"))
 	}
-	if cd.FSType == "" || (cd.FSType != "ext3" && cd.FSType != "ext4") {
-		allErrs = append(allErrs, validation.NewFieldRequired("fsType required and should be of type ext3 or ext4"))
+	if cd.FSType == "" {
+		allErrs = append(allErrs, validation.NewFieldRequired("fsType"))
```

Comment:
I would do that in separate PR, it's 'omitempty' in almost all volume plugins, while the validator for all of them requires it's presence.